src/explore-education-statistics-admin/src/pages/release/footnotes/components/FootnoteSubjectSelection.tsx

```diff
@@ -126,9 +132,25 @@ const FootnoteSubjectSelection = ({
     );
   }
 
+  function renderSubjectItem(
+    { id, selectionType, label }: SubjectItem,
+    children?: ReactNode,
+  ) {
+    return (
+      <li key={id}>
+        {selectionType === 'All' || selectionType === 'Specific' ? (
+          <strong>{label}</strong>
+        ) : (
+          label
```

Comment:
Will this not print out the HTML string "label"?  Not sure :)

Reply:
If I wrap it in curly braces I get this error:
![image](https://user-images.githubusercontent.com/44812484/113017951-edb30400-9177-11eb-840c-3fcb20ae1600.png)

So I assume its correct as it is